hack/run-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+source hack/common.sh
+
+prefix=kubevirt
+tag=v0.5.0-alpha.1
```

Comment:
so each time kubevirt version changes, we will need to update in multiple places? here, glide &  ansible (for the deployment). i wonder how are we going to enforce it :-/

Reply:
@nellyc do you prefer to parse it from `glide.yaml`?

Reply:
thinking about it again, maybe it is good to have the deployment version separate from the tests version, this way you only have 1 moving part at a time. 

and actually when the tests will be containerized, this will get solved by itself, right?

Reply:
ignore my last comment, it was stupid :-/